pandas/tests/test_strings.py

```diff
@@ -420,6 +420,16 @@ def test_str_cat_categorical(self, box, dtype_caller, dtype_target, sep):
             result = s.str.cat(t, sep=sep)
             assert_series_or_index_equal(result, expected)
 
+    @pytest.mark.parametrize('box', [Series, Index, np.array, list])
+    def test_str_cat_raise_wrong_dtype(self, box):
+        # GH 22722
+        s = Series(['a', 'b', 'c', 'd'])
+        t = box([1, 2, 3, 4])
+
+        msg = 'Can only concatenate list-likes containing only strings.*'
```

Comment:
is the .* needed?

Reply:
I like to end regexes for long warning messages with `.*` to make it abundantly clear that the message actually continues.